samples/StandaloneApp/Program.cs

```diff
@@ -1,21 +1,19 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using Microsoft.AspNetCore.Blazor.Browser.Rendering;
-using Microsoft.AspNetCore.Blazor.Browser.Services;
+using Microsoft.AspNetCore.Blazor.Hosting;
 
 namespace StandaloneApp
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            var serviceProvider = new BrowserServiceProvider(configure =>
-            {
-                // Add any custom services here
-            });
-
-            new BrowserRenderer(serviceProvider).AddComponent<App>("app");
+            CreateHostBuilder(args).Build().Run();
         }
+
+        public static IBrowserHostBuilder CreateHostBuilder(string[] args) =>
```

Comment:
As a general thought about terminology, I wonder if we should say `IWebAssemblyHostBuilder` (etc) for all the concepts that are specific to running in the browser on WebAssembly.

That might be a clearer distinction to make rather than calling those "browser" apps, since even the remote-rendered ones run in the browser to some extent.

The "browserness" that I was meaning to capture in the `.Browser` project was more to do with rendering on a browser DOM, as opposed to rendering in memory for server-side prerendering or unit tests. I think that's orthogonal to whether the .NET part is running on the server/electron for remote rendering or under WebAssembly.

Reply:
As discussed, I'll rename all of these